app/forms/idv/api_image_upload_form.rb

```diff
@@ -249,6 +249,14 @@ def validate_images
           type: :not_a_file
```

Comment:
I tried this locally and it worked! 🎉 I did have to make sure these two flags were set:

```
  doc_auth_selfie_capture_enabled: true
  doc_auth_selfie_desktop_test_mode: false
```

I also saw some strange behavior that was perhaps a fluke or the gift card I was using. When I first tried using the SDK after failing the upload I saw:

- only tap to capture as an option
- a few "image is blurry" failures
- second time I was able to use autocapture instead of tap to capture and I'm not clear why
- the selfie autocapture was very slow and strange for me

Again these are probably something I did somehow as I don't see a cause for these issues in this code. But I wanted to note here in case it is a pattern.

Reply:
yes... you're right on the FF settings to trigger a failure 👍🏿